src/style.css

```diff
@@ -0,0 +1,21 @@
+* {
+  margin: 0;
+  padding: 0;
+}
+
+.block {
+  width: 100%;
```

Comment:
no need to write this style, `div` already have it.

Reply:
done:
![image](https://user-images.githubusercontent.com/32244670/61871340-3f0de900-aee9-11e9-8037-633880ef1385.png)